src/tasks/AndroidAppBuilder/Templates/MonoRunner.java

```diff
@@ -34,7 +34,7 @@ public class MonoRunner extends Instrumentation
 {
     static {
         // loadLibrary triggers JNI_OnLoad in these libs
-        System.loadLibrary("System.Security.Cryptography.Native.Android");
+        System.loadLibrary("%JNI_LIBRARY_NAME%");
         System.loadLibrary("monodroid");
```

Comment:
this one doesn't make sense in library mode right?

Reply:
It's still necessary because the only difference in library mode is that it's linked into the library.